typescript/infra/helm/warp-routes/templates/_helpers.tpl

```diff
@@ -58,6 +58,9 @@ The warp-routes container
 - name: warp-routes
   image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
   imagePullPolicy: IfNotPresent
+  env:
```

Comment:
This is a temporary measure because otherwise pino doesn't work :( once we move away from pino in #3582 we should revisit this and move back to json structured logging